gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_4_0/02_federated_apikeys.yml

```diff
@@ -2,8 +2,9 @@ databaseChangeLog:
   - changeSet:
       id: 4.4.0_federated_apikeys
       author: GraviteeSource Team
+      runOnChange: true
       changes:
         - addColumn:
             tableName: ${gravitee_prefix}keys
             columns:
-              - column: {name: federated, type: boolean, constraints: { nullable: true }, defaultValue: false }
```

Comment:
Shouldn't create an another change in 4.4.2? I don't know if re-running the migration that was already run will work. If you have tried I'm good ;)

Reply:
It was a question I wanted to ask. If I remember it well the 4.x is totally not usable for customers using jdbc 🤔 
And I may be wrong but during my tests I seen that boolean null value is possible only for mssql. But I can test it